spec/models/question_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Question, type: :model do
+  context "relationships" do
+    it { should belong_to(:user) }
+    it { should belong_to(:category) }
+    it { should has_many(:answers) }
```

Comment:
I'm not really familiar with shoulda-matchers, but is it supposed to be 'has_many' or 'have_many' for these assertions. I'm thinking the latter just because...grammar, but I'm uncertain.